curriculum/challenges/english/25-front-end-development/lecture-css-specificity-the-cascade-algorithm-and-inheritance/672b8ea434ceac23cc90f337.md

```diff
@@ -34,18 +35,6 @@ ID selectors have a very high specificity, higher than type selectors and class
 
 This means that ID selectors can override class selectors and type selectors but can be overridden by inline styles.
 
-ID selectors can be combined with other selectors to create even more specific rules.
```

Comment:
Here is the issue for this change

https://github.com/freeCodeCamp/freeCodeCamp/issues/60384